### DISCUSSION TITLE
Collection 'user' with 'username' field presentable.

### DISCUSSION THREAD

**Discussion by Karlstens** [pocketbase/pocketbase#5145] [Q&A] [answered]
Hello,

I've started tinkering with pocketbase. Very interesting project. 

I notice as I create a collection, and associate the user collection via a relation field, that it presents the 'users' collection using the email field. 

Is it possible to set the lookup to return the 'username' as the presentable field instead of the email field?

**Comment by ganigeorgiev** [comment, marked as answer]
No, the system fields at the moment cannot be explicitly marked as "Presentable".
This will be possible with the [refactoring](https://github.com/pocketbase/pocketbase/discussions/4355).
